docs/content/preview/yugabyte-cloud/managed-automation/managed-cli.md

```diff
@@ -0,0 +1,697 @@
+: Name of the sink.
+
+--new-name=_name_
+: New name for the sink.
+
+--username=_name_
+: Sink user name.
+
+--password=_password_
+: Sink user password.
+
+#### delete
+
+--name=_name_
+: Name of the sink.
+
+-----
+
+### cdc-stream
+
+Use the `cdc-stream` resource to create, update, and delete CDC streams.
+
+```text
+Usage: ybm cdc-stream [command] [flags]
+```
+
+Examples:
+
+- Create a CDC stream:
+
+  ```sh
+  ybm cdc-stream create
+      --cluster-name=cluster-1 
+      --name=stream-2 
+      --tables=table1,table2 
+      --sink=mysink 
+      --db-name=mydatabase 
+      --snapshot-existing-data=true 
+      --kafka-prefix=prefix
+  ```
+
+#### create
+
+--cluster-name=_name_
+: Name of the cluster with the tables you want to stream.
+
+--name=_name_
+: Name for the stream.
+
+--tables=_table names_
+: List of tables the CDC stream will listen to.
+
+--sink=_sink_
+: Destination sink for the stream.
+
+--db-name=_database name_
+: Database that the Cdc Stream will listen to.
+
+--snapshot-existing-data=_bool_
+: Whether to snapshot the existing data in the database.
+
+--kafka-prefix=_prefix_
+: Prefix for the Kafka topics.
+
+#### get
+
+--cluster-name=_name_
+: Name of the cluster with the streams you want to fetch.
+
+--name=_name_
+: Name of the CDC stream.
+
+#### update
+
+--cluster-name=_name_
+: Name of the cluster with the tables you want to stream.
+
+--name=_name_
+: Name of the stream.
+
+--tables=_table names_
+: List of tables the CDC stream will listen to.
+
+--new-name=_name_
+: New name for the stream.
+
+#### delete
+
+--cluster-name=_name_
+: Name of the cluster with the stream to delete.
+
+--name=_name_
+: Name of the stream.
+
+-----
+
+### backup
+
+Use the `backup` resource to perform operations on cluster backups, including the following:
+
+- create and delete cluster backups
+- restore a backup
+- get information about clusters
+
+```text
+Usage: ybm backup [command] [flags]
+```
+
+Examples:
+
+- Create a backup:
+
+  ```sh
+  ybm backup create 
+      --cluster-name=test-cluster
+      --credentials=username=anonymous,password=password123
+  ```
+
+#### create
+
+--cluster-name=_name_
+: Name of the cluster to back up.
+
+--retention-period=_days_
+: Retention period for the backup in days.
+
+--description=_description_
+: A description of the backup.
+
+#### delete
+
+--backup-id=_id_
+: The ID of the backup to delete.
+
+#### get
+
+--cluster-name=_name_
+: Name of the cluster of which you want to view the backups.
+
+#### restore
+
+--cluster-name=_name_
+: Name of the cluster to restore to.
+
+--backup-id=_id_
+: The ID of the backup to restore.
+
+## Environment variables
+
+In the case of multi-node deployments, all nodes should have similar environment variables.
+
+The following are combinations of environment variables and their uses:
+
+- `YBM_APIKEY`
+
+  The API key to use to authenticate to your YugabyteDB Managed account.
+
+- `YBM_HOST`
+
+  The YugabyteDB Managed host.
+
+## Examples
```

Comment:
This is good for now. However, I test all the commands, provide more examples and update this doc if needed.